Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MapboxMobileEvents",
+    platforms: [
+        .iOS(.v10)
```

Comment:
Does it matter which version of `iOS` we set here?

Reply:
Yes, good catch. Lowered to v9.